File: bezierdemo/src/main/java/com/qi/study/MainActivity.java
package com.qi.study;

import android.app.Activity;
import android.os.Bundle;
import android.os.Handler;
import android.os.HandlerThread;
import android.os.Message;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;
import android.widget.Toast;

import com.qi.study.bezierview.SplitView;

public class MainActivity extends Activity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        final SplitView splitView = (SplitView) findViewById(R.id.splitview);
        findViewById(R.id.dongdong).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
             //   splitView.setRightVrticalLineX( 1.5f);
                splitView.startSplitAnimator();
            }
        });

    }


}

File: amination/src/main/java/com/qi/amination/MainActivity.kt
package com.qi.amination

import android.app.Activity
import android.os.Bundle
import android.view.View
import android.view.animation.Animation
import android.widget.TextView

import com.qi.amination.amination.CustomAmination

class MainActivity : Activity(), View.OnClickListener {

    private var mTextview: TextView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initView()
    }

    private fun initView() {
        mTextview = findViewById(R.id.textviwe) as TextView
        mTextview!!.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        val customAmination = CustomAmination()
        customAmination.setAnimationListener(object : Animation.AnimationListener {
            override fun onAnimationStart(animation: Animation) {

            }

            override fun onAnimationEnd(animation: Animation) {

            }

            override fun onAnimationRepeat(animation: Animation) {

            }
        })
        mTextview!!.startAnimation(customAmination)

    }
}
